test/lib/usd/pxrUsdPreviewSurface/testPxrUsdPreviewSurfaceExportImportRoundtrip.py

```diff
@@ -0,0 +1,138 @@
+#!/pxrpythonsubst
+#
+# Copyright 2020 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pxr import Gf
+from pxr import Sdf
+from pxr import Usd
+from pxr import UsdShade
+from pxr import UsdUtils
+
+import os
+import unittest
+
+from maya import cmds
+from maya import standalone
+
+
+class testPxrUsdPreviewSurfaceExport(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        standalone.initialize('usd')
+
+        test_dir = os.path.join(os.path.abspath('.'),
+                                "PxrUsdPreviewSurfaceExportTest")
+
+        cmds.workspace(test_dir, o=True)
+
+    @classmethod
+    def tearDownClass(cls):
+        standalone.uninitialize()
+
+    @unittest.skip("Needs investigation on MacOS")
```

Comment:
This test is crashing on MacOS...same if I try to import assets with materials.
I disabled the test since we gave it to QA to play with. Please re-enable.